tests/postgres_tests/test_aggregates.py

```diff
@@ -110,6 +110,24 @@ def test_string_agg_empty_result(self):
         values = AggregateTestModel.objects.aggregate(stringagg=StringAgg('char_field', delimiter=';'))
         self.assertEqual(values, {'stringagg': ''})
 
+    def test_string_agg_distinct_false(self):
```

Comment:
It would be fine to put this in a separate test class so it can have a `setUpTestData` method to avoid the delete() and redundant object creation